index.d.ts

```diff
@@ -245,7 +350,7 @@ export interface DailyCallStaticUtils {
 
 export interface DailyCall {
   iframe(): HTMLIFrameElement | null;
-  join(properties?: DailyCallOptions): Promise<DailyParticipant[] | void>;
+  join(properties?: DailyCallOptions): Promise<DailyParticipantsObject>;
```

Comment:
based on our usage locally I think this is actually an object instead of an array of objects. Also, the `| void` is likely not necessary (unless it should be `| null` ?)